test_runner/fixtures/neon_cli.py

```diff
@@ -30,6 +30,7 @@
     T = TypeVar("T")
 
 
+# ruff: noqa: B024
```

Comment:
I have never used ABCs. Should I just remove it as superclass, but the class name as-is? Or do we need ABC as superclass, maybe to prevent instantiation?

Reply:
Yeah, I'd probably remove `abc.ABC` and keep the name as is.
And would add a comment that it used to be an ABC, but the linter [wasn't happy](https://docs.astral.sh/ruff/rules/abstract-base-class-without-abstract-method/), ..etc..)

Reply:
Done. Thank you!